src/tensor_ops.jl

```diff
@@ -256,6 +256,23 @@ end
 Base.ctranspose(S::AllTensors) = transpose(S)
 
 
+############################
+# Symmetric/Skew-symmetric #
+############################
+
+"""
+Computes the symmetric part of a second order tensor.
+"""
+@inline sym{dim,T}(S1::Tensor{2,dim,T}) = convert(SymmetricTensor{2,dim,T},S1)
+@inline sym(S1::SymmetricTensor{2}) = S1
+
+"""
+Computes the skew-symmetric (anti-symmetric) part of a second order tensor.
+"""
+@inline skw(S1::Tensor{2}) = 0.5*(S1 - S1.')
+@inline skw{dim,T}(S1::SymmetricTensor{2,dim,T}) = zero(Tensor{2,dim,T})
+
```

Comment:
Because a skew symmetric tensor is not symmetric? This is just a special case, that will be used in case the tensor is in fact symmetric. Just feels strange that a skew function would now return a skew matrix.